qabelbox/src/main/java/de/qabel/qabelbox/persistence/SimpleJSONModel.java

```diff
@@ -0,0 +1,23 @@
+package de.qabel.qabelbox.persistence;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public abstract class SimpleJSONModel {
```

Comment:
Couldn't the whole JSONModel concept be substituted by a factory and serializer?